osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/MainCirclePiece.cs

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Bindables;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Rulesets.Objects.Drawables;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Rulesets.Osu.Objects.Drawables.Pieces
+{
+    public class MainCirclePiece : CompositeDrawable
```

Comment:
Is there a reason why you didn't combine `CirclePiece` and `MainCirclePiece`, along with making `DrawableSpinner` do a `SkinnableDrawable` lookup? It's very confusing as it is right now: MainCirclePiece -> CirclePiece -> DefaultCirclePiece...

Reply:
Yes, all these can be cleaned up going forward. If you'd like I can do a quick pass now.

Reply:
Well the reason I asked is because I want the resource string to be `Play/osu/hitcircle`, and not the fully-namespaced version it currently is. But that would conflict with what is already in `CirclePiece`...

Reply:
The `CirclePiece` one can be removed. The new implementation replaces it.

Reply:
As I thought, please do then.